### PR TITLE
Adding bootstrap.sh file

### PR DESCRIPTION
**Pull request by FaezZangariya**
Fix issue #3

### CODE REVIEW

**Review by RonTuretzky — COMMENTED** [review]
@ofirtzabari I think these are the changes that I was talking about in your PR 

@kobihk I think this needs to be merged after @ofirtzabari 's PR

**Review by EliasAndry — DISMISSED** [review]

File: setup.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+cd /vagrant
+# Install dependencies with Pipenv
+pipenv sync --dev
+
+# run our app. setsid, the parentheses and "&" are used to perform a "double
+# fork" so that out app stays up after the setup script finishes.
+# The app logs are redirected to the `runserver.log` file.
+(setsid pipenv run \
+    python manage.py runserver 0.0.0.0:8000 > runserver.log 2>&1 &)
```
Comment (kobihk) [inline setup.sh:17]:
```suggestion
# run our app. Nohup and “&” are used to let the setup script finish
# while our app stays up. The app logs will be collected in nohup.out
nohup pipenv run python manage.py runserver 0.0.0.0:8000 &

```

File: Vagrantfile [hunk 2]
```diff
@@ -0,0 +1,81 @@
+# -*- mode: ruby -*-
```
Comment (kobihk) [inline Vagrantfile:1]: After we will merge #6 you will need to rebase your PR 
Then this file probably will be redundant

**Review by kobihk — CHANGES_REQUESTED** [review]

**Comment by kobihk** [comment]
@FaezZangariya please rename the file from setup.sh to bootstrap.sh
as you asked in the work division details:
`Student2: bootstrap.sh`

**Comment by kobihk** [comment]
@FaezZangariya please update the title of this PR from:
"Adding Setup.sh and making changes to Vagrantfile"
to:
"Adding bootstrap.sh file"
and please ask for CR from your teammates.

**Review by RonTuretzky — APPROVED** [review]
LGTM!

**Review by EliasAndry — APPROVED** [review]

**Review by kobihk — APPROVED** [review]

**Pull request merged**